stream_alert/rule_processor/parsers.py

```diff
@@ -90,21 +90,24 @@ def matched_log_pattern(self, record, log_patterns):
                 return self.matched_log_pattern(record[field], pattern_list)
 
             if not isinstance(pattern_list, list):
-                LOGGER.debug('designated log_patterns should be a \'list\'')
+                LOGGER.debug('Configured `log_patterns` should be a \'list\'')
                 continue
 
-            # the pattern field value in the record
+            # The pattern field value in the record
             try:
                 value = record[field]
             except (KeyError, TypeError):
-                LOGGER.debug('declared log pattern field [%s] is not a valid field '
+                LOGGER.debug('Declared log pattern field [%s] is not a valid type '
                              'for this record: %s', field, record)
                 continue
-            # append the result of any of the log_patterns being True
+            # Append the result of any of the log_patterns being True
             pattern_result.append(any(fnmatch(value, pattern)
                                       for pattern in pattern_list))
 
-        LOGGER.debug('%s pattern result: %s', self.type(), pattern_result)
+        all_patterns_result = all(pattern_result)
+        LOGGER.debug('%s log pattern match result: %s', self.type(), all_patterns_result)
+        # if not all_patterns_result:
+            # LOGGER.debug('Failed Log Pattern Match: %s', json.dumps(record, indent=4))
 
         # if all pattern group results are True
         return all(pattern_result)
```

Comment:
change this return to be `return all_patterns_result` since you already do the `all(..)` call above